_posts/pt-BR/pages/2016-03-21-rss.md

```diff
@@ -0,0 +1,10 @@
+---
+title: RSS Feeds
+name: rss
+permalink: /pt-BR/rss/
+type: pages
+layout: page
+lang: pt-BR
+version: 1
+---
+{% include pages/rss.html %}
```

Comment:
I think this line includes a localized page that fails to pass tests as it references untranslated keys.